specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/apimdocumentations.json

```diff
@@ -0,0 +1,363 @@
+      "get": {
+        "tags": [
+          "Documentation"
+        ],
+        "operationId": "Documentation_ListByService",
+        "description": "Lists all Documentations of the API Management service instance.",
+        "x-ms-examples": {
+          "ApiManagementListApis": {
+            "$ref": "./examples/ApiManagementListDocumentations.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Paged Result response of Documentations.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/DocumentationCollection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "./definitions.json#/definitions/DocumentationContract"
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/documentations/{documentationId}": {
+      "head": {
+        "tags": [
+          "Documentation"
+        ],
+        "operationId": "Documentation_GetEntityTag",
+        "description": "Gets the entity state (Etag) version of the Documentation by its identifier.",
+        "x-ms-examples": {
+          "ApiManagementHeadDocumentation": {
+            "$ref": "./examples/ApiManagementHeadDocumentation.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/DocumentationIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
>200 [](http://example.com/codeflow?start=11&length=3)

Here, 200 response only has headers. Please add response schema to be returned.

Reply:
the head request doesn't return a body, just the eTag header. This also reflects in the example `ApiManagementHeadDocumentation` and this is how all our "head" requests are made

Reply:
Got it.